cclib/parser/gaussianparser.py

```diff
@@ -314,6 +314,67 @@ def extract(self, inputfile, line):
         # Basis set name
         if line[1:15] == "Standard basis":
             self.metadata["basis_set"] = line.split()[2]
+            
+        # Solvent information.
+        # PCM (the default gaussian solvent method).
+        if line[1:34] == "Polarizable Continuum Model (PCM)":
+            # Solvent parameters are unique to each solvent model,
+            # so they are packaged together to prevent clogging the namespace.
+            self.metadata['solvent_params'] = {
+            }
+            
+            # Keep looking until dashed only.
+            while set(line.strip()) != set("-"):
+                line = next(inputfile)
+                
+                # PCM has a few different subtypes.
+                # Model                : PCM.
+                if "Model" in line:
+                    self.metadata['solvent_model'] = " ".join(line.split()[2:])[:-1]
+                    
+                    if self.metadata['solvent_model'] == "PCM":
+                        self.metadata['solvent_model'] = "IEFPCM"
+                
+                # Don't think this is necessary...    
+                # elif "Atomic radii" in line:
+                #    self.metadata['solvent_params']['radii'] = " ".join(line.split()[3:])[:-1]
+                elif "Atomic radii" in line and line.split()[-1] == "SMD-Coulomb.":
+                    self.metadata['solvent_model'] = "SMD-IEFPCM"
+                    
+                # Solvent by keyword.
+                #  Solvent              : Toluene, Eps=   2.374100 Eps(inf)=   2.238315
+                # Solvent by definition.
+                #  Solvent              : Generic,
+                #            Eps                           =   9.000000
+                #            Eps(infinity)                 =   2.000000
+
+                elif "Solvent" in line and "Eps=" in line and "Eps(inf)= " in line:
+                    # Capture the human readable name, as well as params.
+                    self.metadata['solvent_name'] = line.split()[2][:-1]
```

Comment:
```suggestion
                    self.metadata['solvent_name'] = line.split()[2][:-1].lower()
```
Also, save `line.split()` in a variable and reuse it for this block.

Reply:
Note to self: it's better to post-process this in the superclass' `after_parsing` so all parsers get it, once the super call is set up.